src/Components/Samples/BlazorUnitedApp/App.razor

```diff
@@ -0,0 +1,42 @@
+﻿@implements IRazorComponentApplication<App>
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <base href="/" />
+    <link rel="stylesheet" href="css/bootstrap/bootstrap.min.css" />
+    <link href="css/site.css" rel="stylesheet" />
+    <link href="BlazorUnitedApp.styles.css" rel="stylesheet" />
+    <link rel="icon" type="image/png" href="favicon.png" />
+    <HeadOutlet></HeadOutlet>
+</head>
+<body>
+    <Router AppAssembly="@typeof(App).Assembly">
+        <Found Context="routeData">
+            <RouteView RouteData="@routeData" DefaultLayout="@typeof(MainLayout)" />
+            <FocusOnNavigate RouteData="@routeData" Selector="h1" />
+        </Found>
+        <NotFound>
+            <PageTitle>Not found</PageTitle>
+            <LayoutView Layout="@typeof(MainLayout)">
+                <p role="alert">Sorry, there's nothing at this address.</p>
+            </LayoutView>
+        </NotFound>
+    </Router>
+
+    <div id="blazor-error-ui">
+        <environment include="Staging,Production">
+            An error has occurred. This application may no longer respond until reloaded.
+        </environment>
+        <environment include="Development">
+            An unhandled exception has occurred. See browser dev tools for details.
+        </environment>
+        <a href="" class="reload">Reload</a>
+        <a class="dismiss">🗙</a>
+    </div>
+
+    <script src="_framework/blazor.web.js" suppress-error="BL9992"></script>
+</body>
+</html>
```

Comment:
With the updated project structure, the split between `App.razor` and `_Layout.razor` seems mostly arbitrary, as any of the content could go in either of them. We could even eliminate `_Layout.razor` entirely if we wanted, putting all its content into `App.razor`.

There could be some practical benefits to moving as much of the content as possible into `_Layout.razor`, though. If `App.razor` was *just* the router, then `_Layout.razor` could contain the entire HTML structure from `<html>` to `</html>`. That would have some benefits:

 * Just one place to look to see the entire common markup - no mental effort to stitch together `App.razor` and `_Layout.razor` to imagine what the full structure looks like
 * People who wanted entirely different HTML in different layouts could easily do so. For example, they could have different pages reference different CSS files in the header, etc.

I haven't checked to verify for sure this fully works end-to-end, and there's no need to block this PR on it. I'm just commenting so that we consider this when we implement the real unified project template. cc @javiercn.

Reply:
I will take this into consideration when I start building the template.